inductiva/tasks/run_simulation.py

```diff
@@ -1,16 +1,22 @@
 """Functions for running simulations via Inductiva Web API."""
 import pathlib
 from typing import Any, Optional
+import json
+
+from absl import logging
 
 from inductiva import tasks, resources, types
 from inductiva.api import methods
+from inductiva.utils import format_utils
 
 
 def run_simulation(
     api_method_name: str,
     input_dir: pathlib.Path,
     machine_group: Optional[resources.MachineGroup] = None,
     storage_dir: Optional[types.Path] = "",
+    save_sim_metadata_dir: Optional[types.Path] = None,
+    extra_sim_metadata_to_save: Optional[dict] = None,
```

Comment:
for now let's not have any additional argument to these methods.
Let's assume that they get called with everything that is strictly required. If it becomes imperative to have additional, custom, metadata fed into these calls, then we'll look into it.